src/fidesops/ops/schemas/connection_configuration/connection_config.py

```diff
@@ -112,7 +154,28 @@ class SaasConnectionTemplateValues(BaseModel):
     secrets: connection_secrets_schemas  # For ConnectionConfig
     instance_key: FidesOpsKey  # For DatasetConfig.fides_key
 
+    class Config:
+        """Add examples for each field"""
+        schema_extra = {
+            "example": {
+                "name": "Foo",
+                "key": FidesOpsKey("eb65d06059de1c30aa2b973a33075"),
+                "description": "A Schema with values to create both a Saas ConnectionConfig and DatasetConfig from a template.",
+                "secrets": PostgreSQLSchema(username="Mahmoud", password="S3c11R3P@ssW0rd", _required_components=["host", ]),
```

Comment:
Here also, something like:

`username="<your_username>", password="<your_password>"` would be better.